src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php

```diff
@@ -38,6 +39,10 @@ public function process(ContainerBuilder $container)
 
             $class = $container->getParameterBag()->resolveValue($definition->getClass());
             if (!is_subclass_of($class, 'Symfony\\Component\\Console\\Command\\Command')) {
+                if (!class_exists($class, false)) {
+                    throw new InvalidArgumentException(sprintf('Class "%s" used for service "%s" cannot be found.', $class, $id));
```

Comment:
we should use `\InvalidArgumentException` for consistency in this PR (and rely on your other PR to use "DI" exceptions)